packages/terra-date-input/src/DateInput.jsx

```diff
@@ -565,7 +597,8 @@ class DateInput extends React.Component {
           onFocus={this.handleMonthFocus}
           onBlur={this.handleMonthBlur}
           disabled={this.props.disabled}
-          aria-describedby={ariaDescriptionId}
+          aria-invalid={this.props.isInvalid}
+          aria-required={this.props.required}
```

Comment:
Not sure if it is out of scope for this change. Should we also add `aria-disabled` if `this.props.disabled` is true?

Reply:
Yes, it should. I will add that.

Reply:
Fixed with 0c9deda2a90b10bb13dcd44a55b11f8d203f86ef